### PR TITLE
Fix implementation according to the latest changes

### PR DESCRIPTION
**Pull request by Sammers21**

### CODE REVIEW

**Comment by g4s8** [comment]
@0crat in

**Comment by 0crat** [comment]
I'm not managing `artipie/vertx-server` GitHub repository, you have to contact me in Slack first, see [§11](http://www.zerocracy.com/policy.html#11)

File: pom.xml [hunk 1]
```diff
@@ -82,7 +82,7 @@ SOFTWARE.
     <dependency>
       <groupId>com.artipie</groupId>
       <artifactId>http</artifactId>
-      <version>0.1.0</version>
+      <version>1.0-SNAPSHOT</version>
```
Comment (g4s8) [inline pom.xml]: @Sammers21 please don't use snapshot versions, use `0.2` versions instead

**Review by g4s8 — CHANGES_REQUESTED** [review]
@Sammers21 just one comment

**Review by g4s8 — APPROVED** [review]

**Comment by Sammers21** [comment]
@rultor, release tag=`0.1`

**Comment by Sammers21** [comment]
@rultor, release tag=`0.1`

**Comment by g4s8** [comment]
@rultor release, tag=`0.1`

**Pull request merged**